pkg/mountutil/process_split.go

```diff
@@ -0,0 +1,100 @@
+//go:build freebsd || linux
```

Comment:
This file should be named `mountutil_others.go` or `mountutil_linux.go` to stay consistent that this a function that is specific to runtimes related to mountutil.

Reply:
If we pass "none" as fstype for windows, it gives this error: "time="2022-05-12T12:13:13+05:30" level=fatal msg="invalid OCI spec - Type 'none' not supported: unknown"".

Reply:
why are we passing nullfs or none, when we are already passing mount types like bind, volume or tmps while splitting the source and destination.

Reply:
These comments look to have landed on the wrong thread. I've responded here: https://github.com/containerd/nerdctl/pull/924#discussion_r871565970